interfaces/system_key_test.go

```diff
@@ -80,7 +80,11 @@ func (s *systemKeySuite) TearDownTest(c *C) {
 	dirs.SetRootDir("/")
 }
 
-func (s *systemKeySuite) testInterfaceWriteSystemKey(c *C, nfsHome bool) {
+func (s *systemKeySuite) testInterfaceWriteSystemKey(c *C, nfsHome, overlayRoot bool) {
```

Comment:
The two booleans are pretty opaque in tests below. Not a priority but I would flip that to some kind of flags argument instead.